block.json

```
@@ -67,5 +67,6 @@
  			"default": []
  		}
  	},
-	"editorScript": "file:build/index.js"
+	"editorScript": "file:build/index.js",
+	"script": "file:build/frontend.js"
```

Same here:
`maps-block-apple-frontend`

```suggestion
	"script": "maps-block-apple-frontend"
```